test/helper.rb

```diff
@@ -1,3 +1,9 @@
+require "simplecov"
+SimpleCov.start do
+  add_filter "/test/"
+  add_filter "/vendor/"
```

Comment:
Nope, didn't change. Maybe on Circle it would have (this uses Travis)? Anyway, good call 👍